routes/web.php

```diff
@@ -16,3 +17,8 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+Route::get('register', [AuthController::class, 'register'])->name('register');
+Route::post('signup', [AuthController::class, 'signup'])->name('singup');
+Route::get('login', [AuthController::class, 'login'])->name('signin');
+Route::post('signin', [AuthController::class, 'signin'])->name('signin');
```

Comment:
@aaldemeery These are two routes with different paths, yet with the same name!! Change the name